server/sonar-server/src/main/resources/org/sonar/server/user/ws/current-example.json

```diff
@@ -14,7 +14,6 @@
     "global": ["profileadmin", "scan"]
   },
   "homepage": {
-    "type": "PROJECT",
-    "value": "project-key"
+    "type": "MY_PROJECTS"
```

Comment:
why have you removed the value field in the response?

Reply:
because generated uuid and of little value here